docs/user-guide/faq.md

```diff
@@ -13,6 +13,12 @@ Please see README.md in the for the release by clicking on the tag for the relea
 ### How do I run the setup on my laptop?
 Follow the [guide on the wiki](https://github.com/vmware/docker-volume-vsphere/wiki/Using-laptop-for-running-the-entire-stack)
 
+### How do I bind a volume to a particular container host?
+This can be achieved via [Tenancy](http://vmware.github.io/docker-volume-vsphere/documentation/tenancy.html).
+
+### Can I migrate data between Linux and Windows containers?
+Docker currently doesn't support migration of binaries between Windows and Linux, so container migration isn't possible. Also, volumes created via the Linux plugin are ext4 formatted by default, while the ones created via the Windows plugin are NTFS formatted, and they cannot be cross-mounted.
```

Comment:
binaries migration is not supported, but the question is not about binaries - it is about data migration. So while this `volumes created via the Linux plugin are ext4 formatted by default, while the ones created via the Windows plugin are NTFS formatted,` is accurate, this `they cannot be cross-mounted.` is not. *if* a Linux host has NTFS driver, e.g. http://www.tuxera.com/community/open-source-ntfs-3g/, they can mount the volume on Linux and use local volumes to provide the data into containers. Similarly, if they have ext4 support on Windows (e.g. http://www.ext2fsd.com/) they can mount the disk and then expose it to containers via local driver. What we want to say "while it is possible, our code does not support it automatically nor provides any explicit  help for these cases"